### DISCUSSION TITLE
[4.x]: Improve UI for Entry fields with many relations

### DISCUSSION THREAD

**Discussion by shifuma** [craftcms/cms#13174] [Ideas] [answered]
### Description
Now that the Entrification plan is underway, it would be great to see some options to better manage how an Entries field with many relations appears. This field used to be Tags, so it was much more manageable.

![screenshot_2023_05_09 _22 11](https://github.com/craftcms/cms/assets/86767543/7ed38af8-16fe-434a-a3d1-3a6223d08189)

I understand this is something that might be coming in Craft 5, but I didn't see any mention on [the blog post](https://craftcms.com/blog/entrification) so I thought it was worth creating an issue.

### Craft CMS version

4.4 or later

**Comment by maxstrebel** [comment]
To add to this, same is for fields with only one relation. As an idea, I could imagine that you can select a display-mode on the relation field. For example display as Tags / List / Dropdown

**Comment by wmdhosting** [comment]
+1 this is overdue.. definitly.. you have allready tag field.. 
Just add it like option to to this field....

**Comment by alexanderbuergin** [comment]
+1 We need this option!

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.8.0 is out with inline search for relation fields, and inline entry creation for Entries fields (#17497) 🎉

**Reply by maxstrebel** [reply]
Hey Brandon, thank you so much for the great improvements in 5.8.0. But I'd see this specific issue not solved yet, as the relations still stack on top of each other, which is not always optimal (https://github.com/craftcms/cms/discussions/17496).

I'd suggest a layout dropdown that let's you choose between "list" and "tags".

**Comment by brandonkelly** [comment]
Craft 5.9 is adding a new “Inline list” view mode for relational fields (https://github.com/craftcms/cms/pull/17744) 🎉

**Reply by alexanderloewe** [reply]
Love that!

**Reply by brandonkelly** [reply]
Craft 5.9.0 is out with that change 🎉
